README.md

```diff
@@ -1,61 +1,53 @@
 <img width="100" align="right" src="img/hermitcore_logo.png" />
 
-# libhermit-rs - A Rust-based library operting system
+# RustyHermit: libhermit-rs
 
 [![Build Status](https://travis-ci.com/hermitcore/libhermit-rs.svg?branch=master)](https://travis-ci.com/hermitcore/libhermit-rs)
 ![Actions Status](https://github.com/hermitcore/libhermit-rs/workflows/Build/badge.svg)
 [![License](https://img.shields.io/crates/l/rusty-hermit.svg)](https://img.shields.io/crates/l/rusty-hermit.svg)
 [![Slack Status](https://radiant-ridge-95061.herokuapp.com/badge.svg)](https://radiant-ridge-95061.herokuapp.com)
 
-[RustyHermit](https://github.com/hermitcore/rusty-hermit) is a [unikernel](http://unikernel.org) targeting a scalable and predictable runtime for high-performance and cloud computing.
-Unikernel means, you bundle your application directly with the kernel library, so that it can run without any installed operating system.
-This reduces overhead, therefore, interesting applications include virtual machines and high-performance computing.
+_libhermit-rs_ is the kernel of the [RustyHermit](https://github.com/hermitcore/rusty-hermit) unikernel project.
 
-_libhermit-rs_ is the heart of RustyHermit and is the kernel itself.
-The kernel is able to run [Rust](https://github.com/hermitcore/hermit-playground) applications, as well as [C/C++/Go/Fortran](https://github.com/hermitcore/rusty-hermit) applications.
+## Building the kernel
 
-## Prerequisites
+Usually the kernel will be linked as static library to your applications.
 
-The Rust toolchain can be installed from the [official webpage](https://www.rust-lang.org/).
-RusyHermit currently requires the **nightly versions** of the toolchain.
-```sh
-rustup default nightly
-```
+- **Rust applications:** Instructions can be found in the [rusty-hermit](https://github.com/hermitcore/rusty-hermit) repository.
+- **For C/C++ applications:** Instructions can be found in the [hermit-playground](https://github.com/hermitcore/hermit-playground) repository.
+ 
+
+### Standalone static library build
+
+If this does not fit your needs and you want to build the kernel as static library to link afterwards, you need the following:
 
-Further requirements are the source code of the Rust runtime, and llvm-tools:
+The Rust **nightly** toolchain ([official webpage](https://www.rust-lang.org/)), the source code of the Rust runtime, and llvm-tools:
 
 ```sh
-rustup component add rust-src
-rustup component add llvm-tools-preview
+rustup default nightly
```

Comment:
It isn't required to set the nightly as default compiler.

Reply:
(fixed in f751654 )